### DISCUSSION TITLE
A virtual Tasmota device.

### DISCUSSION THREAD

**Discussion by wjohn007** [arendst/Tasmota#20538] [Show and tell]
I created a sample [Berry application](https://github.com/wjohn007/Berry-ExtSensor) for integrating sensors from remote Tasmota controllers.

This creates a new virtual controller that merges the origin and foreign properties.

![plan01](https://github.com/arendst/Tasmota/assets/115527085/91199d5f-6c83-48b6-9193-9995644150e6)

**Comment by barbudor** [comment]
Some people really have fun with Berry 🤣 👏

**Reply by s-hadinger** [reply]
Berry **is** fun

**Reply by wjohn007** [reply]
Yes, I'm having a lot of fun with it.
It's a wonderful tool.

**Reply by Jason2866** [reply]
Great to see Berry found fans!
